_pages/en/it-picture-medium-term.md

```diff
@@ -178,53 +176,62 @@ TODO: Find cloud adoption rates | Duplicate from previous section?
+- making use of cloud technologies like Infrastructure-as-Code and containers, especially during development and testing, to drastically improve IITB's responsiveness to change
+- development teams having access to modern development tools needed to keep pace with industry best practices
+
+IT within ESDC (not just IITB) is able to more quickly address multiple projects throughout the year by implementing a quarterly production release, at the very latest (least?).
 By adhering to an agile deployment methodology and a product management approach, the highest business value is delivered early on in the project and made available to clients after each quarter.
 As the circumstances change, this problem is then either worked on for another iteration (one quarter) or, if the relationship between time committed and business value gained declines compared to other projects competing for resources, then another project is onboarded for the proceeding quarter. See [Quarterly Delivery](./quarterly-delivery.md).
 
-TODO: this describes a process, I would stay on outcome, linking instead to the quarterly-delivery document for more details.
-
-- Development teams have access to modern development tools needed to keep pace with industry best practices.
+TODO: This describes a process, I would stay on outcome, linking instead to the quarterly-delivery document for more details.
 
 ##### Internal IM/IT services by IITB
 
 IITB serves as a steward of IM/IT in ESDC and has embedded IT expertise in program areas (since it has moved to DevOps and Product Management).
-As a steward, IT provides ESDC Programs and their employees with the following services:
+As a steward, IITB provides ESDC Programs and their employees with the following services:
 
 - Corporate services (hardware and software support)
 - Cyber security
 - Technology architecture
 - Development teams
 - Technology Infrastructure Management
 
-Corporate Services includes the establishment and promotion of sound Information Management (IM) structure and culture. Information is seen/viewed/treated as a corporate asset and managed through a centralized Records Management System where information is easily searchable, secure and not duplicated.
+[IITB`s Way Forward](http://dialogue/grp/IITB-DGIIT-Gov-New-Nouveau/Documents/Departmental%20IMIT%20Plan/19-20%20Plans/IITB%20Moving%20Forward%20v2.docx) enabled IITB to provide ESDC with modern Corporate Services which includes the establishment and promotion of sound Information Management (IM) structure and culture. Information is seen/viewed/treated as a corporate asset and managed through a Records Management ecosystem where information is easily searchable, secure and not duplicated. IITB fully supports the release of open datasets. Also, collaboration platforms are easily accessible and linked between each other to provide a Digital Workspace experience.
```

Comment:
We probably need to review this section to have 1 sentence per line (easier tracking of changes afterwards)